foundations/introduction/join_the_odin_community.md

```diff
@@ -1,5 +1,5 @@
 ### Introduction
-Working and collaborating with other people is an important part of working as a web developer. Therefore, we at The Odin Project encourage you to participate in our online chat community, which we'll talk more about below. By joining we can all grow together and help each other learn web development. While you're at it, you can check out our [Facebook page](https://www.facebook.com/theodinproject/) and [follow us on Twitter](https://twitter.com/TheOdinProject) and on [Instagram](https://www.instagram.com/theodinproject/).
+Working and collaborating with other people is an important part of working as a web developer. Therefore, we at The Odin Project encourage you to participate in our online chat community, which we'll talk more about below. By joining we can all grow together and help each other learn web development. While you're at it, you can check out our [Facebook page](https://www.facebook.com/theodinproject/) and [follow us on Twitter](https://twitter.com/TheOdinProject) and on [Instagram](https://www.instagram.com/theodinproject/). Use #TheOdinProject to share your Odin Project progress, updates, thoughts and to see what other Odin students are up to!
```

Comment:
Can we use commas here? 
> While you're at it, you can check out our Facebook page, follow us on Twitter, and on Instagram.

Additionally, since we're using verb + noun phrasings for the first two clauses, maybe alter the above to say "and catch up on Instagram".

Reply:
Done!